src/OrchardCore.Modules/OrchardCore.Forms/Views/Form-FlowPart.cshtml

```diff
@@ -0,0 +1,32 @@
+@using OrchardCore.ContentManagement
+@using OrchardCore.Flows.Models
+@using OrchardCore.Flows.ViewModels
+@using OrchardCore.Forms.Models;
+@using OrchardCore.Mvc.Utilities
+
+@model FlowPartViewModel
+@inject OrchardCore.ContentManagement.Display.IContentItemDisplayManager ContentItemDisplayManager
+
+@{
+    var formPart = Model.FlowPart.ContentItem.As<FormPart>();
+}
+
+<form class="flow flow-form" action="@formPart.Action" method="@formPart.Method">
+    @Html.AntiForgeryToken()
```

Comment:
FYI this is not necessary when you are using the form tag helper. But here you are not.

Reply:
Good to know.